src/app/locator/qgslocatoroptionswidget.cpp

```diff
@@ -332,26 +330,30 @@ void QgsLocatorFiltersModel::commitChanges()
     if ( !activePrefix.isEmpty() && activePrefix != filter->prefix() )
     {
       filter->setActivePrefix( activePrefix );
-      settings.setValue( QStringLiteral( "locator_filters/prefix_%1" ).arg( filter->name() ), activePrefix, QgsSettings::Section::Gui );
```

Comment:
Why is this code disabled?

Reply:
Ops that slipped in sorry, will re enable it